chihuahua/assetlist.json

```diff
@@ -206,6 +206,33 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/corso.png"
         }
       ]
+    },
+    {
+      "description": "BackBone Labs Liquid Staked HUAHUA",
+      "denom_units": [
+        {
+          "denom": "cw20:chihuahua1jz5n4aynhpxx7clf2m8hrv9dp5nz83k67fgaxhy4p9dfwl6zssrq3ymr6w",
+          "exponent": 0
+        },
+        {
+          "denom": "bhuahua",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "chihuahua1jz5n4aynhpxx7clf2m8hrv9dp5nz83k67fgaxhy4p9dfwl6zssrq3ymr6w",
+      "base": "cw20:chihuahua1jz5n4aynhpxx7clf2m8hrv9dp5nz83k67fgaxhy4p9dfwl6zssrq3ymr6w",
+      "name": "BackBone Labs Liquid Staked HUAHUA",
+      "display": "bHUAHUA",
```

Comment:
In all cases, there seems to be a misunderstanding of the "display" [denom] property. It is meant to reference one of the defined denom_units, and in all your additions, the value for display is not defined. I recommend making the display values fully lowercase (since that's how it's defined in denom_units)

Reply:
okay, ill fix these things.

Reply:
```json
{
      "description": "BackBone Labs Liquid Staked WHALE",
      "denom_units": [
        {
          "denom": "factory/migaloo1mf6ptkssddfmxvhdx0ech0k03ktp6kf9yk59renau2gvht3nq2gqdhts4u/boneWhale",
          "exponent": 0
        },
        {
          "denom": "boneWHALE",
          "exponent": 6
        }
      ],
      "base": "factory/migaloo1mf6ptkssddfmxvhdx0ech0k03ktp6kf9yk59renau2gvht3nq2gqdhts4u/boneWhale",
      "name": "boneWHALE",
      "display": "boneWHALE",
      "symbol": "bWHALE",
      "logo_URIs": {
        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/bWHALE.png"
      },
      "images": [
        {
          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/bWHALE.png"
        }
      ]
    }
```

so this one is ok because the `denom_units[1].denom` matches `display`?  i was following the pattern of others which put the 2nd denom in the denom_units array as the last part of the contract address.